src/types/annotator.js

```diff
@@ -13,6 +18,36 @@
  * @prop {DocumentMetadata} document
  */
 
+/**
+ * An object representing the location in a document that an annotation is
+ * associated with.
+ *
+ * @typedef Anchor
+ * @prop {AnnotationData} annotation
+ * @prop {HTMLElement[]} [highlights]
+ * @prop {Range} [range]
+ */
+
+/**
+ * Anchoring implementation for a particular document type (eg. PDF or HTML).
+ *
+ * This is responsible for converting between serialized "selectors" that can
+ * be stored in the Hypothesis backend and ranges in the document.
+ *
+ * @typedef AnchoringImpl
+ * @prop {(root: HTMLElement, selectors: Selector[], options: any) => Promise<Range>} anchor
+ * @prop {(root: HTMLElement, range: Range, options: any) => Promise<Selector[]>} describe
+ */
+
+/**
+ * Subset of the annotator `Guest` instance that is exposed to other modules
```

Comment:
_That_ is what "Annotator" is?! Hah. Laughing all the way to the...nah, I'm going nowhere.

Reply:
Yes. I guess it made more sense in the context of the "Annotator JS" project. In future this could be refactored to have the object passed as a constructor argument.